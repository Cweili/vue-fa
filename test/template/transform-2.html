<p>
  <fa :icon="faMagic" :scale="1.2" size="4x" style="background: mistyrose"/>
  <fa :icon="faMagic" :scale="0.5" :translateX="0.2" size="4x" style="background: mistyrose"/>
  <fa :icon="faMagic" :scale="0.5" :translateY="0.2" size="4x" style="background: mistyrose"/>
</p>
