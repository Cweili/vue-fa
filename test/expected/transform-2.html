<p><svg viewbox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,0) scale(1.2,1.2) ">
        <path d="M14.06 463.3C-4.686 444.6-4.686 414.2 14.06 395.4L395.4 14.06C414.2-4.686 444.6-4.686 463.3 14.06L497.9 48.64C516.6 67.38 516.6 97.78 497.9 116.5L116.5 497.9C97.78 516.6 67.38 516.6 48.64 497.9L14.06 463.3zM347.6 187.6L452.6 82.58L429.4 59.31L324.3 164.3L347.6 187.6z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewbox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(102.4,0) scale(0.5,0.5) ">
        <path d="M14.06 463.3C-4.686 444.6-4.686 414.2 14.06 395.4L395.4 14.06C414.2-4.686 444.6-4.686 463.3 14.06L497.9 48.64C516.6 67.38 516.6 97.78 497.9 116.5L116.5 497.9C97.78 516.6 67.38 516.6 48.64 497.9L14.06 463.3zM347.6 187.6L452.6 82.58L429.4 59.31L324.3 164.3L347.6 187.6z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg><svg viewbox="0 0 512 512" class="vue-fa" style="height:1em;font-size:4em;vertical-align:-.125em;transform-origin:center;overflow:visible;background:mistyrose;" aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg">
    <g transform="translate(256 256)" transform-origin="128 0">
      <g transform="translate(0,102.4) scale(0.5,0.5) ">
        <path d="M14.06 463.3C-4.686 444.6-4.686 414.2 14.06 395.4L395.4 14.06C414.2-4.686 444.6-4.686 463.3 14.06L497.9 48.64C516.6 67.38 516.6 97.78 497.9 116.5L116.5 497.9C97.78 516.6 67.38 516.6 48.64 497.9L14.06 463.3zM347.6 187.6L452.6 82.58L429.4 59.31L324.3 164.3L347.6 187.6z" fill="currentColor" transform="translate(-256 -256)"></path>
      </g>
    </g>
  </svg></p>